common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -345,28 +359,20 @@ def answer_problem(self, correct):
             self.problem_page.click_choice("choice_1")
 
     @attr('shard_7')
```

Comment:
@alisan617 this syntax to specify a shard is not correct-- look at the other test points.

This test point is running and passing, but in shard 9, which is where all test points that don't specify a shard runs.

https://build.testeng.edx.org/job/edx-platform-test-subset/555400/console